server/app.ts

```diff
@@ -61,7 +62,8 @@ export default function createApp(
   recipientFormService: RecipientFormService,
   zendeskService: ZendeskService,
   cjsmService: CjsmService,
-  supportedPrisonsService: SupportedPrisonsService
+  supportedPrisonsService: SupportedPrisonsService,
```

Comment:
this is why I disagree with the typescript template that doesn't allow commas at the end of functions - https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/.eslintrc.json#L93